src/RenderPropsComponents/WeekDay/test.tsx

```diff
@@ -57,90 +59,80 @@ test('trigger callback on mouseenter', () => {
   expect(handleOnEnter).toBeCalled()
 })
 
-test('custom render', () => {
```

Comment:
No more if else, every sub component will be rendered by a render prop, and on the top most component, we will set these as default props